docling_core/transforms/serializer/markdown.py

```diff
@@ -282,9 +329,24 @@ def _serialize_image_part(
                 text_res = image_placeholder
             else:
                 text_res = f"![Image]({str(item.image.uri)})"
+            
+        elif image_mode == ImageRefMode.DESCRIPTION:
```

Comment:
`ImageRefMode` is not supposed to be used for switching to the description output.
We recently introduced the concept of Serializers exactly for this purpose.

See docs at: https://docling-project.github.io/docling/concepts/serialization/
Examples at: https://docling-project.github.io/docling/examples/serialization/